core/benches/fs_vs_monoiofs_vs_compfs/README.md

```diff
@@ -0,0 +1,13 @@
+# OpenDAL services fs vs. monoiofs vs. compfs
+
+This benchmark compares the performance of OpenDAL services fs, monoiofs and compfs.
+
+## Goal
```

Comment:
Hi, this isn't the benchmark goal. We shouldn't make assumptions about it. In fact, monoiofs can't outperform tokio-based fs in every case.

Reply:
> In fact, monoiofs can't outperform tokio-based fs in every case.

Actually, monoiofs is expected to be slower since it is currently single threaded. I'm trying to express the final goal we wish to achieve.  Did I misunderstood the point of this section?

Reply:
> Did I misunderstood the point of this section?

Our goal is to measure the performance of various fs services so users can choose for themselves, rather than trying to prove that monoiofs is faster than tokio fs. Please note that OpenDAL does not declare a winner; we simply offer options.

Reply:
Thanks for pointing out that!